src/components/radio-button/radio-button.style.ts

```diff
@@ -9,7 +9,15 @@ import { StyledLabelContainer } from "../../__internal__/label/label.style";
 import baseTheme from "../../style/themes/base";
 import FormFieldStyle from "../../__internal__/form-field/form-field.style";
 
-const RadioButtonStyle = styled(CheckboxStyle)`
+export interface RadioButtonProps {
```

Comment:
For the sake of consistency you should probably import RadioButtonProps from `radio-button.component.tsx` and pick props the ones used by RadioButtonStyle instead of redefining them again